buddyboss/buddyboss-platform
# PROD-722 Fix - Emojis are not fully shown on the site using a mobile device

### All Submissions:


### Changes proposed in this Pull Request:

Fixes #3045 

### Connected PR in theme
https://github.com/buddyboss/buddyboss-theme/pull/1682

### How to test the changes in this Pull Request:

1. Go to activity or group message or message or forum
2. Search emoji popup with Capital letter: "Smile"
3. It should make it lowercase and work fine

### Proof Screenshots or Video
https://www.loom.com/share/02658f4bbc104fc68fa344df6d3090a8

### Other information:
